package.json

```diff
@@ -262,6 +262,7 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-native": "^4.0.0",
     "ethers": "^5.7.2",
+    "expect": "^29.7.0",
```

Comment:
fixes a knip error. We use expect directly in e2e tests. It was previously a transitive dependency from jest which knip doesn't allow https://knip.dev/guides/handling-issues#unlisted-dependencies